### DISCUSSION TITLE
A URL for creating a new discussion with a pre-selected category

### DISCUSSION THREAD

**Discussion by chuanqisun** [community/community#3888] [Discussions]
It'd help me share a link that takes people directly to the right category.

**Comment by abskmj** [comment]
Upvote!

I would also request the functionality to pre-populate the title and template from the URL parameters similar to what is already there for issues `https://github.com/{owner}/{repo}/issues/new?title=foo&body=bar`.

**Comment by thanhcsf** [comment]
Same question.

**Comment by juliettech13** [comment]
Same!

**Comment by Xelliese** [comment]
Same here. It would be much more easier.
